yarn.lock

```diff
@@ -2,18 +2,30 @@
 # yarn lockfile v1
 
 
-"@babel/code-frame@^7.0.0", "@babel/code-frame@^7.12.13":
+"@babel/code-frame@^7.12.13":
```

Comment:
get the latest merged yarn.lock file instead, there should be little to no changes, except perhaps for your jest update